AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/AWSSDKSwiftCLI/Subcommands/PrepareRelease.swift

```diff
@@ -58,53 +58,67 @@ struct PrepareRelease {
         case awslabs = "awslabs"
         case smithyLang = "smithy-lang"
```

Comment:
Much trailing white space & doc comments that should be regular comments are fixed below.